CROSSSECTOR/DIH/DigitalInnovationHubService/CONTRIBUTORS.yaml

```diff
@@ -0,0 +1,30 @@
+description: This is a compilation list of all CONTRIBUTORS across different objects (data models) alphabetically ordered. All fields are non mandatory

```

Comment:
contributors file is only available at the subject level. Therefore it should be one level up and only one file.